### PR TITLE
[Spring Beans] LPS-200638 Register Destination in activate method

### PR DESCRIPTION
**Pull request by JanisZhang**
module: `export-import-service`, `message-boards-service`, `subscription-service`. Thanks.
https://liferay.atlassian.net/browse/LPS-200638

### CODE REVIEW

File: modules/apps/subscription/subscription-service/src/main/java/com/liferay/subscription/internal/messaging/SubscriptionSenderMessageListener.java [hunk 1]
```diff
@@ -28,6 +37,27 @@
 )
 public class SubscriptionSenderMessageListener extends BaseMessageListener {
 
+	@Activate
+	protected void activate(BundleContext bundleContext) {
+		DestinationConfiguration destinationConfiguration =
```
Comment (dantewang) [inline modules/apps/subscription/subscription-service/src/main/java/com/liferay/subscription/internal/messaging/SubscriptionSenderMessageListener.java:42]: @JanisZhang Please inline this variable, also in https://github.com/liferay-core-infra/liferay-portal/pull/6108 and https://github.com/liferay-core-infra/liferay-portal/pull/6107

**Review by dantewang — CHANGES_REQUESTED** [review]

**Comment by JanisZhang** [comment]
ci:test:sf

**Comment by JanisZhang** [comment]
ci:test:relevant

**Comment by JanisZhang** [comment]
ci:test:document

**Comment by JanisZhang** [comment]
ci:help

**Comment by JanisZhang** [comment]
ci:test:message-boards

**Comment by JanisZhang** [comment]
ci:test:export-import

**Review by dantewang — APPROVED** [review]

**Comment by catarineaguiar** [comment]
no regressions found

**Comment by dantewang** [comment]
ci:forward:force

cc @tinatian

**Pull request closed without merging**